### PR TITLE
Music 'playable' ref pages

### PR DESCRIPTION
**Pull request by ganicke**
Add in some reference pages for the new "playable" blocks.

Do we need a page for the Song Editor like micro:bit has for its [Sound Editor](https://makecode.microbit.org/types/sound#sound-editing)?

Closes https://github.com/microsoft/pxt-arcade/issues/5599

### CODE REVIEW

**Review by Jaqster — APPROVED** [review]
Looks good.  Thanks Galen!

**Comment by Jaqster** [comment]
I do think it would be good to have a separate docs page for how to use the Song Editor, although music notation is more of a known/familiar thing than the sound effect stuff is.  But there are some 'hidden' features that would be good to call out.  Can you take a stab at pulling this together Galen?  Thanks!

**Comment by riknoll** [comment]
Some of the hidden things:

1. Ctrl+Click on a note to cycle between sharp+flat
2. The whole thing is keyboard navigable. All the shortcuts are listed here: https://github.com/microsoft/pxt/blob/master/webapp/src/components/musicEditor/keyboardNavigation.ts#L4

File: libs/mixer/docs/reference/music/play.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+music.play(music.createSong(hex`0078000408020200001c00010a006400f40164000004000000000000000000000000000500000430000400080001220c001000012514001800011e1c00200001222400280001252c003000012934003800012c3c004000011e03001c0001dc00690000045e010004000000000000000000000564000104000330000400080001290c001000011e1400180001251c002000012924002800011b2c003000012234003800011e3c0040000129`), music.PlaybackMode.UntilDone)
+```
+
+## Parameters
+
+* **toPlay**: the [playable](/types/playable) object, or music source, to play.
+* **playbackMode**: the playback mode for continuing the program:
+>* `play until done`: play the music source in **toPlay** but wait to run the next part of the program until music play is done.
+>* `in background`: play the music source in **toPlay** but continue with the rest of the program before music play is done.
+>* `in background looping`: play the music source in **toPlay** but continue with the rest of the program before music play is done. The music will remain playing, returning to the first note of the music after its duration.
```
Comment (riknoll) [inline libs/mixer/docs/reference/music/play.md:36]: Might be worth mentioning that you can stop the looping by using the "stop all sounds" block!

**Review by riknoll — APPROVED** [review]

File: libs/mixer/docs/reference/music/play.md (hunk 1 above)
Comment (ganicke) [reply libs/mixer/docs/reference/music/play.md:36]: 👍

**Comment by ganicke** [comment]
Will do a Song Editor page in a separate PR.

**Pull request merged**